paper/paper.Rmd

```diff
@@ -0,0 +1,305 @@
+  index: 2
+- name: Institute of Medical Sociology, University Medical Center Hamburg-Eppendorf, Germany
+  index: 3
+- name: Independent Researcher, Ramat Gan, Israel
+  index: 4
+- name: Independent Researcher, Tampa, FL, USA
+  index: 5
+- name: Center for Humans and Machines, Max Planck Institute for Human Development, Berlin, Germany
+  index: 6
+- name: Department of Psychology, Université du Québec à Montréal, Montréal, Québec, Canada
+  index: 7
+correspondence: D.Makowski@sussex.ac.uk.
+type: article
+date: "`r Sys.Date()`"
+bibliography: paper.bib
+# abstract: |
+#   The `modelbased` package provides a straightforward approach to computing and visualizing model-based estimates in R, including marginal means, contrasts, slopes, and predictions. Designed as part of the easystats ecosystem, it streamlines post-hoc analysis and model interpretation across a wide range of statistical models. While tools like `emmeans` and `marginaleffects` offer similar functionality, `modelbased` prioritizes ease of use, making it more accessible to researchers without extensive statistical programming expertise. By integrating modern workflows for effect estimation and visualization, modelbased enhances the interpretability and reproducibility of statistical analyses in psychological and social sciences.
+keywords: |
+  marginal effects; marginal means; model predictions; R; easystats
+acknowledgement: |
+  *{performance}* is part of the collaborative
+  [*easystats*](https://github.com/easystats/easystats) ecosystem
+  [@easystatspackage]. Thus, we thank all
+  [members of easystats](https://github.com/orgs/easystats/people),
+  contributors, and users alike.
+authorcontributions: |
+  D.M. drafted the paper; all authors contributed to both the writing of the
+  paper and the conception of the software.
+funding: |
+  This research received no external funding.
+conflictsofinterest: |
+  The authors declare no conflict of interest.
+output:
+  rticles::joss_article
+journal:
+  pissn: 2475-9066
+link-citations: true
+csl: apa.csl
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  comment = "#>",
+  out.width = "100%",
+  dpi = 300,
+  warning = FALSE
+)
+
+library(easystats)
+```
+
+# Statement of need
+
+Applied statistics have historically been focused on statistical *tests* (e.g., *t*-tests, correlation tests, ANOVAs), seen as most apt to provide researchers with interpretable answers to the questions they seek.
+However, these tests are typically based on statistical *models*, which are the true underlying cornerstone of modern data science.
+The replication crisis [@OSC2015estimating; @camerer2018evaluating] and methodological revolution(s) [@makowski2023we] have underlined the issues with the traditional focus on statistical tests (the effacement of model assumptions, a focus on null-hypothesis testing, non-compatibility with more complex variance structures), and the need to shift the focus to the models themselves.
```

Comment:
Can we have one or two examples for "the issues"? Doesn't need to be full sentences.